.golangci.yml

```diff
@@ -2,6 +2,11 @@
 run:
   concurrency: 6
   deadline: 5m
+issues:
+  exclude-rules:
+    - path: fake_client\.go
```

Comment:
nit:

Ideally we'd skip all `counterfeiter` generated files / packages. We can do this by
- filename (`fake_interface.go`)
- dir/filename (`somepackagefakes/fake_someinterface.go`)
- package name (`k8s.io/release/somepath/somepackagefakes`)
- file content (`Code generated by counterfeiter. DO NOT EDIT.`)

I will check how to best do this with golangci. But let's not block on that.

Reply:
Awesome, thanks :pray: